internal/cli/roles_permissions.go

```diff
@@ -0,0 +1,268 @@
+package cli
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/spf13/cobra"
+	"gopkg.in/auth0.v5"
+	"gopkg.in/auth0.v5/management"
+)
+
+var (
+	roleAPIIdentifier = Flag{
+		Name:       "API",
+		LongForm:   "api-id",
+		ShortForm:  "a",
+		Help:       "API Identifier.",
+		IsRequired: true,
+	}
+
+	roleAPIPermissions = Flag{
+		Name:       "Permissions",
+		LongForm:   "permissions",
+		ShortForm:  "p",
+		Help:       "Permissions.",
+		IsRequired: true,
+	}
+)
+
+func rolePermissionsCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "permissions",
+		Short: "Manage permissions within the role resource",
+		Long:  "Manage permissions within the role resource.",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listRolePermissionsCmd(cli))
+	cmd.AddCommand(addRolePermissionsCmd(cli))
+	cmd.AddCommand(removeRolePermissionsCmd(cli))
+
+	return cmd
+}
+
+func listRolePermissionsCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID string
+	}
+
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "List permissions defined within a role",
+		Long: `List existing permissions defined in a role. To add a permission try:
+auth0 roles permissions add <role-id>`,
+		Example: `auth0 roles permissions list <role-id>
+auth0 roles permissions ls`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				err := roleID.Pick(cmd, &inputs.ID, cli.rolePickerOptions)
+				if err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			var list *management.PermissionList
+
+			if err := ansi.Waiting(func() error {
+				var err error
+				list, err = cli.api.Role.Permissions(inputs.ID)
+				return err
+			}); err != nil {
+				return fmt.Errorf("An unexpected error occurred: %w", err)
+			}
+
+			cli.renderer.RolePermissionList(list.Permissions)
+			return nil
+		},
+	}
+
+	return cmd
+}
+
+func addRolePermissionsCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID            string
+		APIIdentifier string
+		Permissions   []string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "add",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Add a permission to a role",
+		Long: `Add an existing permission defined in one of your APIs.
+To add a permission try:
+
+    auth0 roles permissions add <role-id> -p <permission-name>`,
+		Example: `auth0 roles permissions add <role-id> -p <permission-name>
+auth0 roles permissions assoc`,
```

Comment:
The example needs to be updated

Reply:
It still says `assoc`, as there's no alias now, the second example should be removed.

Reply:
Sorry missed that. Done!